pom.xml

```diff
@@ -152,6 +152,11 @@
         <artifactId>proto-google-common-protos</artifactId>
         <version>${google.common-protos.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core</artifactId>
+        <version>${google.core.version}</version>
+      </dependency>
```

Comment:
I think while declaration, mentioning the scope is better.

Reply:
I meant this  is root pom.xml, then mentioning the `<scope>test</scope>` here would be better.

Reply:
are you taking about this ? 
https://github.com/pmakani/java-tasks/pull/1/files#diff-54ee0bcb0b1bf83c4d2c0d60e5cad9d5R84-R88

Reply:
I meant the same scope but in root pom.xml instead of sub-module

Reply:
actually dependencies related suggestion by chingor and we are following to him.
so it won't be issue anymore i guess :)